### PR TITLE
README - transition between modes

### PR DESCRIPTION
**Pull request by BAlmeidaS**
Adding a small session on the README trying to instruct others to enable a different transition from Normal mode to Jupyter Command mode only with `Shift+ESC` instead of the default `ESC`

Related to this issue: https://github.com/jupyterlab-contrib/jupyterlab-vim/issues/126

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,13 @@ Shortcuts this extension introduces:
 | Z, M    | Hide All Code Cells |
 | Z, R    | Show All Code Cells |
 
+### Transition between modes
+
+Note that there are three modes (Insert, Normal and Jupyter Command). The transition between them happens only with the key ESC, but you can enable a different transition from Normal mode to Jupyter Command mode.
+
+Go to *Setting / Settings Editor / Notebook Vim* and unmark the option `Enable Esc and Ctrl-[ leaving vim Normal mode to Jupyter Command mode`. From now on, `ESC` will move you from Insert to Normal mode and `Shift+ESC` will move you from Normal to Jupyter Command mode.
```
Comment (krassowski) [inline README.md:110]: Minor formatting suggestion:

```suggestion
Go to *Setting / Settings Editor / Notebook Vim* and unmark the option `Enable Esc and Ctrl-[ leaving vim Normal mode to Jupyter Command mode`. From now on, <kbd>Esc</kbd> will move you from Insert to Normal mode and <kbd>Shift</kbd> + <kbd>Esc</kbd> will move you from Normal to Jupyter Command mode.
```

Comment (krassowski) [reply README.md:110]: Though feel free to disregard though as the rest of the readme is using code markers so maybe not worth it for the sake of consistencty?

Comment (BAlmeidaS) [reply README.md]: I think it was a good recommendation, I changed it :+1:

**Comment by firai** [comment]
Thanks for your contribution. Sorry about the delay in reviewing this. I have some ideas about how I would suggest rewording this, but I haven't had time to type it all out and edit it yet.

In the meanwhile, @ianhi and @krassowski, what do you think about the proposed location for placing the new section? The description about modes is near the top of the README, but the reader would need to scroll through installation instructions and the whole list of other shortcuts before they find the proposed section about mode transition shortcuts in the current PR. I'm wondering whether we should at least put the proposed mode transition shortcut section above the other shortcuts.

**Comment by krassowski** [comment]
Shourtcut (link) sounds good to me.

**Comment by firai** [comment]
Thanks for your contribution, @BAlmeidaS! Sorry about the epic delay in reviewing this. I understand that you were trying to make the section small, but the combination of various options and various modes might confuse new users, so I took the chance to expand your PR to be similar to the vim help page. I also added a link from the earlier section describing the modes, based on the discussion above.
